data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,288 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Take the train to the Arachi game.)`
+					goto arachi
+				`	(Take the train to the cooperative game.)`
+					goto wp
+			
+			label "seen saryd, arachi"
+			choice
+				`	(Take the train to the Kimek game.)`
+					goto kimek
+				`	(Take the train to the cooperative game.)`
+					goto nwp
+			
+			label "seen kimek, arachi"
+			choice
+				`	(Take the train to the Saryd game.)`
+					goto saryd
+				`	(Take the train to the cooperative game.)`
+					goto wp
+			
+			label "seen all"
+			choice
+				`	(Take the last train.)`
+					goto wp
+				
+			
+			label saryd
+			apply
+				set "coalition games: saryd"
+			`	You arrive at a massive, stadium-like structure, only it's actually shaped like a half moon. The place overlooks a huge valley. Many small drones fly over it, transmitting live footage to the dozens of cinema-sized screens for those that can't get a clear view through the glass.`
+			`	You notice starting gates on one of the sides of the track. A few minutes pass before an alarm sounds, and the massive crowd roars. The doors open: several Saryd burst from the gates with wheeled machines, similar to chariots, attached to them. They are also carrying bows and arrows.`
+			`	The contestants scatter among a variety of paths leading into the increasingly dense forest. Targets appear for a few seconds as the Saryds approach, and they attempt to shoot at them. Eventually, you have to turn and look at the massive monitors, as it becomes impossible to follow the race by looking out the window.`
+			`	One participant takes a corner going too fast, and their "chariot" slams into a tree. He slows to a stop and throws his bow into the ground. The drones don't transmit sound, but it's clear from his reaction he's been disqualified. Over the din of the crowd, you hear voices coming through the speakers, likely commentators.`
+			`	As they move deeper, tall brambles and ledges become more common among the path, and the targets appear faster and faster. A Saryd navigates one section perfectly, nailing seven different targets in a row. You notice his picture on another monitor rises several positions, putting him near the front. It seems that hitting more targets counts towards a point system.`
+			`	You weren't able to count how many started, but as the race approaches its end, you're able to see roughly thirty made it through - not many. As the Saryds cross a finish line marked by a stream, the monitors switch to a ground crew interviewing the finishers.`
+			`	It looks like Ulmodi finished fourth; he's up next to be interviewed. The event seems to be over, and the crowd disperses under the supervision of Heliarch agents. Instead of heading back to the train, they're moving to a different section of the oddly-shaped stadium.`
+			`	You go with the crowd and are shown to a room with many elevators. Eventually, everyone descends to the valley below and follows a path to an actual, full-size stadium, and a massive one at that.`
```

Comment:
This would be a good place to put a choice in order to break up these events. Perhaps a Heliarch agent asks you about how you liked the event before sending you on to the next one. Building off of my suggestion on the lines above, there's also some information that needs to be condensed down to here.

I'd organize the choice to be something like this, but you can change these choices to whatever you think fits better.
```suggestion
			`	With the event over, the crowd disperses under the supervision of Heliarch agents. The agents guide the audience to rooms with many large elevators. You board one of the elevators, which begins descending down into the valley. During the descent, a Saryd Heliarch agent approaches you and begins speaking through her translation device. "Enjoyed the event, have you human?"`
			choice
				`	"It was a very entertaining event. I can see why the crowd was excited the whole time."`
				`	"I think the event would have been more interesting had I known what exactly was going on."`
			`	The elevator finishes its descent and the doors open. The Heliarch agent nods at your response. "Hope that you enjoy this next event, I do," she says as she begins guiding the audience along a path leading to the building you could see from above. Upon entering, you find that the massive building is a circular stadium with a rectangular court of some sort in the center. Small mounds dot the stadium grounds, while six large holes are arranged evenly around the sides of the field.`
```